File: CRM/Models/ViewModels/MessageViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CRM.Models.ViewModels
{
    public class MessageViewModel
    {
        public string CustomerId { get; set; }

        public string LeadId { get; set; }

        public int LeadAssignmentId { get; set; }

        public List<RecipientViewModel> Recipients { get; set; }

        public string Subject { get; set; }

        public string Message { get; set; }
    }
}

File: CRM/wwwroot/js/viewmodels/MessageViewModel.js
class MessageViewModel {
    constructor(recipients, subject, message, customerId, leadId, leadAssignmentId) {
        this.Recipients = recipients
        this.Subject = subject;
        this.Message = message;
        this.CustomerId = customerId;
        this.LeadId = leadId;
        this.LeadAssignmentId = leadAssignmentId;
    }
}